### PR TITLE
feat: add posts back, add upgrade post as tester

### PR DESCRIPTION
**Pull request by JacobCoffee**
add posts back
add test post for django upgrade

### CODE REVIEW

File: _posts/2025-07-28-python-org-django-5-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+### Rollout
+
+Ideally, we would have a staging environment to test the upgrade before deploying to production but due 
+to the nature of Python.org's infrastructure, we opt for a direct upgrade on the production service.
+
+This is where having a robust test suite and monitoring eases your nerves 😅.
+
+## Lessons Learned
+
+1. **Comprehensive Testing is mportant**: Our extensive test suite caught numerous edge cases before they reached production.
```
Comment (JacobCoffee) [inline _posts/2025-07-28-python-org-django-5-upgrade.md:73]:
```suggestion
1. **Comprehensive Testing is important**: Our extensive test suite caught numerous edge cases before they reached production.
```
